libs/blocks/locui-create/fragments/view.js

```diff
@@ -0,0 +1,74 @@
+import { findFragments, showFragments } from './index.js';
+import { useEffect, useCallback, html, useState } from '../../../deps/htm-preact.js';
+import { getUrls } from '../../locui/loc/index.js';
+
+function Loader() {
+  return html`<div class='locui-create-loader-container'>
+  <div class='locui-create-loader'></div>
+  <p>Loading</p>
+  </div>`;
+}
+
+export default function FragmentsSection({ fragments, setFragments, urls }) {
+  const [validFragments, setValidFragments] = useState([]);
```

Comment:
Since isValidFragments should restrict the navigation of user to another screen, I suppose it should go in store at some global level

Reply:
Hi Nishant,
This Valid fragments state is not responsible for restricting the user, it only for showing valid fragment path from which the user can select fragments.